### PR TITLE
Adding root link for Hoyle

### PR DESCRIPTION
**Pull request by larsacus**

### CODE REVIEW

File: TeamSnapSDK/SDK/DataTypes/TSDKRootLinks.h [hunk 1]
```diff
@@ -18,6 +18,7 @@
 @property (nonatomic, weak) NSURL *_Nullable linkTeamsResults;
 @property (nonatomic, weak) NSURL *_Nullable linkMe;
 @property (nonatomic, weak) NSURL *_Nullable linkMemberRegistrationSignups;
+@property (nonatomic, weak, nullable) NSURL * linkHoyle;
```
Comment (jrahaim) [inline TeamSnapSDK/SDK/DataTypes/TSDKRootLinks.h]: For quick readability, do we want to keep the definitions, at least in this file, standard? either conforming this to the other definitions `property (nonatomic, weak) NSURL *_Nullable`, or search/replace (in this file) so they all match `@property (nonatomic, weak, nullable) NSURL *`. 

As it is it looks like `linkHoyle` is special.

**Review by jrahaim — CHANGES_REQUESTED** [review]
other than the style question and this is 👍

File: TeamSnapSDK/SDK/DataTypes/TSDKRootLinks.h (hunk 1 above)
Comment (larsacus) [reply TeamSnapSDK/SDK/DataTypes/TSDKRootLinks.h]: I almost changed them all, but decided against it right before committing.

Comment (larsacus) [reply TeamSnapSDK/SDK/DataTypes/TSDKRootLinks.h]: Went ahead and added full nullability to this file

**Comment by jrahaim** [comment]
:+1: Unit Tests
:+1: Code

**Review by jrahaim — APPROVED** [review]

**Pull request merged**